3_visualize.yml

```diff
@@ -25,6 +26,11 @@ targets:
       svg_width = svg_width, 
       svg_height = svg_height)
       
+  src/assets/anomaly_peaks.svg:
+    command: send_to_vue(
+      target_name,
+      in_file = '3_visualize/out/anomaly_peaks.svg')
```

Comment:
not necessary but one final thing is that you don't need a separate function here. You can simply make `file.copy()` the function for the target.

```
src/assets/anomaly_peaks.svg:
    command: file.copy(
        to = target_name,
        from = '3_visualize/out/anomaly_peaks.svg')
```

Reply:
lol totally

Reply:
just need to use `to = ` and `from = ` because `target_name` has to be passed in as the first arg

Reply:
Also need `overwrite = TRUE` in the file.copy function so this works as expected, otherwise it silently doesn't replace the `to` target even when the `from` file has changed. Remember [this issue](https://github.com/USGS-R/slim-data-release-template/commit/7d61fcaecc0b8926fcb2cb16670a0c4942070914) @lindsayplatt ?

Reply:
Apparently I still haven't learned my lesson! Thanks @jread-usgs. They had that in the custom function below but I neglected to carry it over to this suggestion :)